frontend/components/forms/fields/DropdownWrapper/DropdownWrapper.tsx

```diff
@@ -0,0 +1,334 @@
+/**
+ * This is a new component built off react-select 5.4
+ * meant to replace Dropdown.jsx built off react-select 1.3
+ *
+ * See storybook component for current functionality
+ *
+ * Prototyped on UserForm.tsx but added and tested the following:
+ * Options: text, disabled, option helptext, option tooltip
+ * Other: label text, dropdown help text, dropdown error
+ */
+
+import classnames from "classnames";
+import React from "react";
+import Select, {
+  StylesConfig,
+  DropdownIndicatorProps,
+  OptionProps,
+  components,
+  PropsValue,
+  SingleValue,
+} from "react-select-5";
+
+import { COLORS } from "styles/var/colors";
+import { PADDING } from "styles/var/padding";
+
+import FormField from "components/forms/FormField";
+import DropdownOptionTooltipWrapper from "components/forms/fields/Dropdown/DropdownOptionTooltipWrapper";
+import Icon from "components/Icon";
+
+const getOptionBackgroundColor = (state: any) => {
+  return state.isSelected || state.isFocused
+    ? COLORS["ui-vibrant-blue-10"]
+    : "transparent";
+};
+
+export interface CustomOptionType {
+  label: string;
+  value: string;
+  tooltipContent?: string;
+  helpText?: string;
+  isDisabled?: boolean;
+}
+
+export interface IDropdownWrapper {
+  options: CustomOptionType[];
+  value?: PropsValue<CustomOptionType> | string;
+  onChange: (newValue: SingleValue<CustomOptionType>) => void;
+  name: string;
+  className?: string;
+  labelClassname?: string;
+  error?: string;
+  label?: JSX.Element | string;
+  helpText?: JSX.Element | string;
+  isSearchable?: boolean;
+  isDisabled?: boolean;
+}
+
+const baseClass = "dropdown-wrapper";
+
+const DropdownWrapper = ({
+  options,
+  value,
+  onChange,
+  name,
+  className,
+  labelClassname,
+  error,
+  label,
+  helpText,
+  isSearchable,
+  isDisabled = false,
+}: IDropdownWrapper) => {
+  const wrapperClassNames = classnames(baseClass, className);
+
+  const handleChange = (newValue: SingleValue<CustomOptionType>) => {
+    onChange(newValue);
+  };
+
+  // Ability to handle value of type string or CustomOptionType
+  const getCurrentValue = () => {
+    if (typeof value === "string") {
+      return options.find((option) => option.value === value) || null;
+    }
+    return value;
+  };
```

Comment:
If the options list is long, could running this every time the component is re-rendered get expensive? Or do we typically limit the options list? / This is not a concern?